TSC-Charter.md

```diff
@@ -176,10 +187,42 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public with a clear indication given of whether the
+individual voted or abstained.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
+
+Should a vote to remove a member result in a tie (50%/50% split of non
+abstaining participants) the matter shall be referred to a Foundation
+appointed independent third party mediator for arbitration.
+
+At any point prior to calling for a vote, the individual member in question, the
+individuals opening the request, or the TSC Chair may request that the issue
+be referred to a Foundation appointed third party mediator for
+arbitration.
```

Comment:
(comment copied so as to not get lost in the outdated fold)

It says that anyone can request that the issue be handled by a third party mediator instead of by a vote of the TSC, but it's not clear how or if that request is handled.

For example, if I'm on the TSC, but acting like a jackass, I might know that the members of the TSC are going to vote me out. In order to undermine this process, I request that it be referred to a third-party mediator, specifically because that gives me more leverage.

Does the foundation board get to say "No, that request is in bad faith, the TSC is going to vote on this"? And by what criteria would the foundation board make that decision?

I'd suggest removing this loophole. The conditions that lead to mediation should be objective and clear, and not dependent on anyone's request. Either it should always go to mediation, or only in the case of a tie, or never.

Reply:
I will try to drudge up some language/process for this and coordinate with legal on how this needs to work.

Reply:
@hackygolucky thanks that will be very helpful.